src/agent/services/lazer_exporter.rs

```diff
@@ -356,23 +405,13 @@ mod lazer_exporter {
                         payload: Some(buf),
                         special_fields: Default::default(),
                     };
-                    if let Err(e) = relayer_sender.send_price_update(&signed_lazer_transaction).await {
-                        tracing::error!("Error publishing update to Lazer relayer: {e:?}");
-                        bail!("Failed to publish update to Lazer relayer: {e:?}");
-                    }
-                }
-                // Handle messages from the relayers, such as errors if we send a bad update
-                mapped_msg = stream_map.next() => {
-                    match mapped_msg {
-                        Some((relayer_url, Ok(msg))) => {
-                            tracing::debug!("Received message from relayer at {relayer_url}: {msg:?}");
-                        }
-                        Some((relayer_url, Err(e))) => {
-                            tracing::error!("Error receiving message from at relayer {relayer_url}: {e:?}");
-                        }
-                        None => {
-                            tracing::error!("relayer connection closed");
-                            bail!("relayer connection closed");
+                    for relayer_sender in relayer_senders.iter() {
+                        if let Err(e) = relayer_sender
+                            .send(signed_lazer_transaction.clone())
+                            .await
+                        {
+                            tracing::error!("Error sending transaction to Lazer relayer session: {e:?}");
+                            // TODO: Under what circumstances would the channel be hosed and is it worth retry?
```

Comment:
The channels are mpsc so I think the only way to receive error is if the channel is closed.